### PR TITLE
Move completer to deephaven_internal

### PR DESCRIPTION
**Pull request by devinrsmith**
Follow-up to #3114

### CODE REVIEW

File: py/server/deephaven_internal/completer/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 See https://github.com/davidhalter/jedi for information on jedi.
 
 # To disable autocompletion
-from deephaven.completer import jedi_settings
+from deephaven_internal.completer import jedi_settings
```
[the remark this answers is not in the document]
Comment (chipkent) [reply py/server/deephaven_internal/completer/__init__.py:9]: I acutally suggested `autocomplete`

[the remark this answers is not in the document]
Comment (jmao-denver) [reply py/server/deephaven_internal/completer/__init__.py:9]: Another thing we might want to take care of is the enum names in _completer.py, our convention is to use all CAPS (also requested by Chip in the original PR)

**Review by jmao-denver — COMMENTED** [review]
some symbols could be renamed to be consistent.

**Review by jmao-denver — APPROVED** [review]
LGTM

**Pull request merged**